src/index.js

```diff
@@ -15,8 +15,8 @@ const view = (state, actions) => {
     <div>
       <div class="song">
         <div class="play-controls">
-          {!state.playing && <button class="button button-default" onclick={() => actions.resume()}>|&gt;</button>}
-          {state.playing && <button class="button button-default" onclick={() => actions.suspend()}>||</button>}
+          {!state.playing && <button class="button button-default" onclick={() => actions.resume()}><i class="fas fa-play"></i></button>}
+          {state.playing && <button class="button button-default" onclick={() => actions.suspend()}><i class="fas fa-pause"></i></button>}
```

Comment:
We could probably bump these up in size, since these are _probably_ the most prominent actions on the page.

Reply:
Also self-closing tags :)